mlflow/environment_variables.py

```diff
@@ -617,3 +617,9 @@ def get(self):
 MLFLOW_DATABRICKS_ENDPOINT_HTTP_RETRY_TIMEOUT = _EnvironmentVariable(
     "MLFLOW_DATABRICKS_ENDPOINT_HTTP_RETRY_TIMEOUT", int, 500
 )
+
+#: The number of urllib3 connection pools to cache.
```

Comment:
Can we add more of a description to users for what this overrides and when to use it?